Adding missing HPC faculty to Chinese Academy of Sciences

dingwentao opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [X] Combine multiple updates to a single institution into a **single PR.**

- [X] Only submit one pull request per institution.

- [X] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [X] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [X] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [X] Check to make sure that you have no spaces after commas, or any missing fields.

- [X] Check to make sure the home page is correct.

- [X] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [X] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [X] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

Commit:
Updating Da Yan affiliation from UAB to Indiana University

Commit:
add Guangming Tan to Chinese Academy of Sciences

Commit:
add Weile Jia to Chinese Academy of Sciences

Commit:
add Yunquan Zhang to Chinese Academy of Sciences

Commit:
add Ninghui Sun to Chinese Academy of Sciences

Commit:
add Junmin Xiao to Chinese Academy of Sciences

emeryberger commented on csrankings-j.csv:
```diff
@@ -2416,6 +2416,7 @@ Junjun Jiang,Harbin Institute of Technology,http://homepage.hit.edu.cn/jiangjunj
 Junliang Xing,Chinese Academy of Sciences,https://sites.google.com/site/junliangxing,jSwNd3MAAAAJ
 Junlin Lu,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=5937,NOSCHOLARPAGE
 Junlin Zhou,UESTC,http://en.uestc.edu.cn/index.php?m=content&c=index&a=show&catid=79&id=4779,pOmjyCEAAAAJ
+Junmin Xiao,Chinese Academy of Sciences,https://people.ucas.edu.cn/~xiaojunmin,NOSCHOLARPAGE
```
Web page does not indicate that this person can solely advise PhD students.

dingwentao replied:
I've removed him from this PR.

emeryberger commented:
Please address issue in review and correct title of this PR.

Commit:
remove Junmin Xiao

Commit:
add Xuebin Chi to Chinese Academy of Sciences

Commit:
add Zhan Wang to Chinese Academy of Sciences

dingwentao commented:
> Please address issue in review and correct title of this PR.

I've addressed this issue. I also added one more faculty member whose website indicates he can supervise PhD students. Additionally, I changed the title of the PR and closed another redundant PR ([#6607](https://github.com/emeryberger/CSrankings/pull/6607/)). Please review and approve. Thanks!

dingwentao commented on csrankings-z.csv:
```diff
@@ -49,6 +49,7 @@ Zeyu Mi,Shanghai Jiao Tong University,https://ipads.se.sjtu.edu.cn/pub/members/z
 Zeyu Wang 0003,HKUST,https://zachzeyuwang.github.io/,q7NLPG0AAAAJ
 Zeyun Yu,University of Wisconsin - Milwaukee,https://uwm.edu/engineering/people/yu-ph-d-zeyun,9kT9SaIAAAAJ
 Zhan Qin,Zhejiang University,https://person.zju.edu.cn/qinzhan,5fa4lOQAAAAJ
+Zhan Wang,Chinese Academy of Sciences,https://people.ucas.ac.cn/~wangzhan,NOSCHOLARPAGE
```
I removed him from this PR as well due to the disambiguation entry in DBLP.

Commit:
remove Zhan Wang

dingwentao commented:
Overall, five HPC people from the Chinese Academy of Sciences—Guangming Tan, Weile Jia, Yunquan Zhang, Ninghui Sun, and Xuebin Chi, all qualified to supervise PhD students—were requested to be added to CSrankings in this PR.
